integreat_cms/cms/views/pages/page_tree_view.py

```diff
@@ -81,13 +84,46 @@ def get(self, request, *args, **kwargs):
                 "language_tree",
                 **{
                     "region_slug": region.slug,
-                }
+                },
             )
 
         if not request.user.has_perm("cms.change_page"):
-            messages.warning(
-                request, _("You don't have the permission to edit or create pages.")
-            )
+            access_granted_pages = Page.objects.filter(
+                Q(editors=request.user) | Q(publishers=request.user)
+            ).filter(region=request.region)
+            if request.user.organization:
+                access_granted_pages = access_granted_pages.union(
+                    Page.objects.filter(organization=request.user.organization)
+                )
+            if len(access_granted_pages) > 0:
+                messages.error(
+                    request,
+                    format_html(
+                        "{}<ul class='list-disc pl-4'>{}</ul>",
+                        _("You can edit only those pages: "),
+                        format_html_join(
+                            "\n",
+                            "<li><a href='{}' class='underline hover:no-underline'>{}</a></li>",
+                            [
+                                (
+                                    translation.backend_edit_link,
+                                    translation.title,
+                                )
+                                for translation in [
+                                    page.get_translation(
+                                        request.region.default_language.slug
+                                    )
+                                    or page.best_translation
+                                    for page in access_granted_pages
+                                ]
```

Comment:
```suggestion
                                    page.best_translation.backend_edit_link,
                                    page.best_translation.title,
                                )
                                for page in access_granted_pages
```
I would directly prefer `best_translation` here - we specifically created this property for this use case, so if the user has the UI in English, the English translation is shown (if it exists), otherwise the default language is used as fallback.